### PR TITLE
Resource manager ID clarifications

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

File: nservicebus/ravendb/resourcemanagerid.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ When using RavenDB in an environment where you are relying also on distributed t
 
 > "A resource manager with the same identifier is already registered with the specified transaction coordinator"
 
-The above is generally due to multiple RavenDB server instances, running on the same machine, trying to enlist in the same transaction with the same resource manager identifier.
+The above is generally due to multiple RavenDB `IDocumentStore` instances, running on the same machine, trying to enlist in the same transaction with the same resource manager identifier.
+
+When using NServiceBus with the RavenDB persistence a constant and deterministic ResourceMagagerId is automatically generated for each endpoint. If you are configuring the endpoint to use your own `IDocumentStore` instance please be sure to setup the store with a valid resource manager identifier, other then the default one.
```
Comment (SimonCropp) [inline nservicebus/ravendb/resourcemanagerid.md]: > If you are configuring the endpoint to use your own `IDocumentStore` instance please be sure to setup the store with a valid resource manager identifier, other then the default one.

how about 

> When configuring the endpoint to use a `IDocumentStore` instance ensure that it has a valid resource manager identifier, ie it **does not** use the default one.

**Comment by SimonCropp** [comment]
@indualagarsamy could you qa this

File: nservicebus/ravendb/resourcemanagerid.md (hunk 1 above)
Comment (mauroservienti) [inline nservicebus/ravendb/resourcemanagerid.md]: Much better. Updated, thanks

**Comment by SimonCropp** [comment]
@mauroservienti do u think this needs to be illustrated with some code??

File: nservicebus/ravendb/resourcemanagerid.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ When using RavenDB in an environment where you are relying also on distributed t
 
 > "A resource manager with the same identifier is already registered with the specified transaction coordinator"
 
-The above is generally due to multiple RavenDB server instances, running on the same machine, trying to enlist in the same transaction with the same resource manager identifier.
+The above is generally due to multiple RavenDB `IDocumentStore` instances, running on the same machine, trying to enlist in the same transaction with the same resource manager identifier.
+
+When using NServiceBus with the RavenDB persistence a constant and deterministic ResourceMagagerId is automatically generated for each endpoint. When configuring the endpoint to use a `IDocumentStore` instance ensure that it has a valid resource manager identifier, ie it does not use the default one.
```
Comment (SimonCropp) [inline nservicebus/ravendb/resourcemanagerid.md]: perhaps bold "does not"??

File: nservicebus/ravendb/resourcemanagerid.md [hunk 3]
```diff
@@ -10,7 +10,9 @@ When using RavenDB in an environment where you are relying also on distributed t
 
 > "A resource manager with the same identifier is already registered with the specified transaction coordinator"
 
-The above is generally due to multiple RavenDB server instances, running on the same machine, trying to enlist in the same transaction with the same resource manager identifier.
+The above is generally due to multiple RavenDB `IDocumentStore` instances, running on the same machine, trying to enlist in the same transaction with the same resource manager identifier.
```
Comment (SimonCropp) [inline nservicebus/ravendb/resourcemanagerid.md]: > The above is generally due to multiple RavenDB `IDocumentStore` instances that, when running on the same machine, attempt to enlist in the same transaction with matching resource manager identifiers.

File: nservicebus/ravendb/resourcemanagerid.md (hunk 2 above)
Comment (mauroservienti) [inline nservicebus/ravendb/resourcemanagerid.md]: fixed

File: nservicebus/ravendb/resourcemanagerid.md (hunk 3 above)
Comment (mauroservienti) [inline nservicebus/ravendb/resourcemanagerid.md]: fixed

**Comment by mauroservienti** [comment]
@SimonCropp 

> do u think this needs to be illustrated with some code??

Other then the snippet? Don't know if you should show how to generate or to use a constant ID

**Comment by janovesk** [comment]
I changed the wording slightly @mauroservienti. Take a look.

**Comment by mauroservienti** [comment]
Looks good @janovesk thanks. What do you think about more code samples/snippets as proposed by @SimonCropp? I'd say that as a user I'd love to a end-to-end guidance on how to solve the issue that I'm facing

**Comment by SimonCropp** [comment]
Merging this one. It is incrementally better and the snippet can be added as another piece of work

**Pull request merged**